cfme/configure/configuration/server_settings.py

```diff
@@ -0,0 +1,409 @@
+    """
+    CONSOLE_TYPES = ('VNC', 'VMware VMRC Plugin', 'VMware WebMKS')
+
+    basic_information = ['company_name', 'appliance_name', 'appliance_zone', 'time_zone', 'locale']
+    server_roles = ('embedded_ansible', 'ems_metrics_coordinator', 'ems_operations',
+                    'ems_metrics_collector', 'reporting', 'ems_metrics_processor', 'scheduler',
+                    'smartproxy', 'database_operations', 'smartstate', 'event', 'user_interface',
+                    'web_services', 'ems_inventory', 'notifier', 'automate',
+                    'rhn_mirror', 'database_synchronization_role', 'git_owner', 'websocket',
+                    'storage_metrics_processor', 'storage_metrics_collector',
+                    'storage_metrics_coordinator', 'storage_inventory', 'vmdb_storage_bridge',
+                    'cockpit_ws')
+    vmware_console = ['console_type']
+    ntp_servers = ['ntp_server_1', 'ntp_server_2', 'ntp_server_3']
+    smtp_server = ['host', 'port', 'domain', 'start_tls', 'ssl_verify', 'auth', 'username',
+                   'password', 'from_email', 'to_email']
+    web_services = ['mode', 'security']
+    logging = ['log_level']
+    custom_support_url = ['url', 'description']
+
+    pretty_attrs = ['appliance']
+
+    def __init__(self, appliance, **kwargs):
+        self.appliance = appliance
+        full_form = (self.basic_information + list(self.server_roles) + self.vmware_console +
+                     self.ntp_servers + self.smtp_server + self.web_services + self.logging +
+                     self.custom_support_url)
+        for att in full_form:
+            setattr(self, att, None)
+        for name, value in kwargs.items():
+            if name == 'console_type':
+                if name not in self.CONSOLE_TYPES:
+                    raise ConsoleTypeNotSupported(value)
+                if self.appliance.version < '5.8':
+                    raise ConsoleNotSupported(
+                        product_name=self.appliance.product_name,
+                        version=self.appliance.version
+                    )
+            # embedded_ansible available from 5.8 version
+            elif name != 'embedded_ansible' and self.appliance.version > '5.7':
+                setattr(self, name, value)
+
+
+# ============================= Basic Information Form =================================
+
+    def update_basic_information(self, updates, reset=False):
+        """ Navigate to a Server Tab. Updates basic information form
+        Args:
+             updates: dict, widgets will be updated regarding updates.
+                ex. update_basic_information({'company_name': 'New name'})
+             regarding updates.
+             reset: By default(False) changes will not be reset, if True changes will be reset
+        """
+        view = navigate_to(self, 'Details')
+
+        view.basic_information.fill(updates)
+        self._save_action(view, updates, reset)
+
+    @property
+    def basic_info(self):
```

Comment:
**REQUIRED:** If we change the name above, this can be called ```basic_information```

Reply:
Updated + _form in the end)